pkg/network/protocols/http/ebpf_main.go

```diff
@@ -340,7 +341,7 @@ func (e *ebpfProgram) init(buf bytecode.AssetReader, options manager.Options) er
 
 	//	If Kafka monitoring is not enabled, loading the program will cause a verifier issue and should be avoided.
 	if !e.cfg.EnableKafkaMonitoring {
-		options.ExcludedFunctions = []string{"socket__kafka_filter"}
+		options.ExcludedFunctions = []string{"socket__kafka_filter", "socket__protocol_dispatcher_kafka"}
 	}
```

Comment:
consider merging it with lines 378-380 so we can have a single location with all changes

Reply:
Done